include/SFML/Graphics/Transform.hpp

```diff
@@ -350,7 +349,7 @@ constexpr Vector2f operator*(const Transform& left, const Vector2f& right);
 } // namespace sf
 
 
-#endif // SFML_TRANSFORM_HPP
+
```

Comment:
You probably want to keep only 2 blank lines here like it is done in other files.

Reply:
Oh I was ninja'd by @Radix-Comet :smile:

Reply:
Thank you both @kimci86 @Radix-Comet; that was a lot to go through!

I'll address it soon.